externs/shaka/player.js

```diff
@@ -1332,6 +1333,10 @@ shaka.extern.ManifestConfiguration;
  *   The minimum time between recoveries when VIDEO_ERROR is reached, in
  *   seconds.
  *   Defaults to <code>5</code>.
+ * @property {boolean} useActualLiveStreamDuration
+ *   If <code>true</code>, the media source live duration
+ *   set as a<code>Infinity<code>
+ *   Defaults to <code> false </code>.
```

Comment:
@joeyparrish Should we set it by default to true in modern browsers? perhaps detecting the support in some way? Maybe checking if https://www.w3.org/TR/media-source/#dom-mediasource-setliveseekablerange is supported?

Reply:
Maybe using: `MediaSource.prototype.setLiveSeekableRange `, if defined defaults true, otherwise false

Reply:
@joeyparrish can you provide your suggestion?

Reply:
Even the Chrome version older than 62 version supports the duration as positive infinity, we can't conclude based on the `MediaSource.prototype.setLiveSeekableRange` alone. We hope our proposed PR will be an optimal solution to set media source live duration as positive infinity. 

CC: @avelad @joeyparrish

Reply:
We have been waiting for this fix to be released since it's a P1 issue for us.  Upon release, could you please backport to the 4.3.x release branch. Because we noticed that [#6070](https://github.com/shaka-project/shaka-player/issues/6070) has reported memory constraints on the web tv. 
CC: @avelad @joeyparrish

Reply:
> We have been waiting for this fix to be released since it's a P1 issue for us. Upon release, could you please backport to the 4.3.x release branch. Because we noticed that [#6070](https://github.com/shaka-project/shaka-player/issues/6070) has reported memory constraints on the web tv. CC: @avelad @joeyparrish

This PR is marked as "feat", which can't be backported.  Only backward-compatible bugfixes can be cherry-picked to a release branch.  New features are released in minor-version bumps only.

I feel that "feat" is appropriate, given that it adds new configs to the system.

Reply:
Sure. Thanks @joeyparrish